platform/ios/src/MGLMapboxEvents.m

```diff
@@ -12,8 +12,8 @@
 
 #include <sys/sysctl.h>
 
-static const NSUInteger version = 1;
```

Comment:
What’s the difference between `version` and the version substring in `MGLMapboxEventsUserAgent`? Why was it possible for them to differ until now?